monitor/service.go

```diff
@@ -64,8 +107,12 @@ func New(c Config) *Monitor {
 func (m *Monitor) Open() error {
 	m.Logger.Printf("Starting monitor system")
```

Comment:
As a nit, and I'm not sure how to get around this yet, for testing, you can't silence this as it will "Open" from the server before you can call "SetLogger" on it to null it out.  I get a lot of chatter in my test output because of it.  I would like to find a better way to do this.